### DISCUSSION TITLE
Set local password on Oauth created accounts

### DISCUSSION THREAD

**Discussion by vmstan** [writefreely/writefreely#634] [Questions / Support]
I currently run write.vmst.io as an extended benefit of a Mastodon instance. I only want users with accounts on Mastodon to be able to login and create blogs on my WriteFreely instance. I've currently accomplished this with Oauth.

My problem is users cannot use tools like the writeas-cli since it requires a local user password.

Is there an incantation of the config.ini that would only allow account creations through Oauth but let those created accounts provide an optional local password?

**Comment by thebaer** [comment]
Hey, this is a great point that we should definitely discuss and possibly improve. I'm going to convert this to a discussion, as the issue tracker is [only for reporting bugs](https://github.com/writefreely/writefreely/blob/develop/CONTRIBUTING.md).
